### ISSUE TITLE
Add block.timestamp to log and events

### ISSUE THREAD

**Issue by 3esmit** [ether-camp/ethereum-sandbox#73]
I think that would be useful for my tests.

So before `Sandbox Event` could be concatenated the time: `[block.timestamp] Sandbox Event` 
Maybe disableable by config.

I know I could edit all my events and include block.timestamp, but that's not handy.

![image](https://cloud.githubusercontent.com/assets/224810/23506796/cc4514ec-ff28-11e6-98e6-060356a89563.png)

**Comment by romanman** [comment]
@3esmit : what is the rational to include `block.timestamp` into that log

**Comment by 3esmit** [comment]
It's useful for debugging when I relay on timestamp in my contract. Without this I need to take note time I deployed, time I called certain function. This way would be easier to check timing functions.

**Comment by romanman** [comment]
@3esmit yeah, but you know I can think about 10-15 more options to trace there, don't you think it can be over burden on the log output logic
